docs/4.4/preview/teleport-application-access.md

```diff
@@ -0,0 +1,206 @@
+# Teleport Application Access Beta
+
+Teleport is currently beta testing an application access proxy. Teleport Application Access has been designed to secure internal web applications, letting you provide secure access while improving both visibility and control for access.
```

Comment:
Are we still beta testing now or should this be GA as of 5.0?

Reply:
4.4. is preview, 5.0 will not have this page.

Reply:
Removed the 5.0 copy, and addressed all other comments.